rosidl_generator_py/rosidl_generator_py/generate_py_impl.py

```diff
@@ -12,9 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from ast import literal_eval
+import keyword
 import os
 import pathlib
+import sys
+from ast import literal_eval
```

Comment:
I'm surprised this is at the end of the list, but as long as the linters are happy it's fine.

Reply:
As I expected the linter doesn't care for this:

https://ci.ros2.org/job/ci_linux/9131/testReport/junit/(root)/projectroot/flake8/